superset/security/manager.py

```diff
@@ -81,7 +81,11 @@ def __init__(self, **kwargs):
 PermissionModelView.list_widget = SupersetSecurityListWidget
 
 # Limiting routes on FAB model views
-UserModelView.include_route_methods = RouteMethod.CRUD_SET | {"userinfo"}
+UserModelView.include_route_methods = RouteMethod.CRUD_SET | {
+    "userinfo",
+    "action_post",
+    "action",
+}
```

Comment:
Should we be using `constants.RouteMethod.ACTION_POST` etc here?

Reply:
oh! yes, right you are